index.html

```diff
@@ -35,5 +35,113 @@ <h2 class="title">I'm a Software Developer</h2>
         <div class="medium"><svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" clip-rule="evenodd"><path d="M12 0c-6.626 0-12 5.372-12 12 0 6.627 5.374 12 12 12 6.627 0 12-5.373 12-12 0-6.628-5.373-12-12-12zm6.974 17.557v-.235l-1.092-1.072c-.096-.073-.144-.194-.124-.313v-7.874c-.02-.119.028-.24.124-.313l1.118-1.072v-.235h-3.869l-2.758 6.88-3.138-6.88h-4.059v.235l1.308 1.575c.128.115.194.285.176.457v6.188c.038.223-.032.451-.189.614l-1.471 1.784v.235h4.17v-.235l-1.471-1.784c-.158-.163-.233-.389-.202-.614v-5.352l3.66 7.985h.425l3.143-7.985v6.365c0 .17 0 .202-.111.313l-1.13 1.098v.235h5.49z"/></svg></div>
       </div>
   </section>
+  <section class="recent-works">
+    <div class="title">
+      <div class="text">My Recent Works</div>
+      <div><img src="media/underline.svg" alt=""></div>
+    </div>
+    <div class="img-card-1">
+      <div class="main-image"><img src="media/Img-Placeholder.png" alt=""></div>
+      <div class="project-title">Multi-Post Stories</div>
+      <div class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</div>
+      <div class="buttons">
+        <div class="simple-button">css</div>
+        <div class="simple-button">html</div>
+        <div class="simple-button">bootstrap</div>
+        <div class="simple-button">Ruby</div>
+        <div class="project-button">See Project</div>
+      </div>
```

Comment:
- [ ] This should be a list `ul` and each of them a `li` and for the rest projects